Runtime/Core/ProBuilderMeshFunction.cs

```diff
@@ -211,7 +211,7 @@ public static ProBuilderMesh Create(IEnumerable<Vector3> positions, IEnumerable<
             return mesh;
         }
 
-        void GeometryWithPoints(Vector3[] points)
+        internal void GeometryWithPoints(Vector3[] points)
```

Comment:
what new code is using this that required it to be internal? my preference would be to deprecate `GeometryWithPoints` in favor of any of the other initialization functions, but if this is just moving code that's already using then it's fine